backend/estimate-service/src/main/java/org/egov/works/web/models/EstimateWrapper.java

```diff
@@ -0,0 +1,18 @@
+package org.egov.works.web.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.*;
+
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+public class EstimateWrapper {
```

Comment:
Where do we need this? The EstimateRequest object has one extra attribute of RequestInfo. Can we reuse that class?

Reply:
ok. I'll make the necessary changes and remove `EstimateWrapper` model class